app/scripts/modules/google/src/loadBalancer/configure/http/httpLoadBalancer.write.service.js

```diff
@@ -33,9 +33,9 @@ angular
       const job = {
         type: 'deleteLoadBalancer',
         loadBalancerName: loadBalancer.listeners[0].name,
-        regions: ['global'],
-        region: 'global',
-        loadBalancerType: 'HTTP',
+        regions: [loadBalancer.region || 'global'],
+        region: loadBalancer.region || 'global',
+        loadBalancerType: loadBalancer.loadBalancerType,
```

Comment:
Sorry, again I'm pretty unfamiliar with this code...but did you make these changes in order for this service to be shared between external and internal HTTP(S) LBs?

Reply:
Yes, I believe sharing these code is easier than other ways at deck...